### PR TITLE
fix: #1593 issue is resolved

### PR DESCRIPTION
**Pull request by NileshPranami**
Changes code lines in onBackPressed() in
SkillsActivity.kt

### CODE REVIEW

File: app/src/main/java/org/fossasia/susi/ai/skills/SkillsActivity.kt [hunk 1]
```diff
@@ -72,14 +72,19 @@ class SkillsActivity : AppCompatActivity(), SkillFragmentCallback {
...
+        if(count==1){
             finish()
             exitActivity(this)
-        }
+        }else if(count==2){
+            title=getString(R.string.skills_activity)
+        }else
+            title=getString(R.string.skills_activity)
+
+
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/skills/SkillsActivity.kt]: Not a good logic

File: build.gradle [hunk 2]
```diff
@@ -7,11 +7,11 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.2'
+        classpath 'com.android.tools.build:gradle:3.2.1'
         classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.2'
         classpath 'com.github.dcendents:android-maven-gradle-plugin:1.4.1'
         classpath "io.realm:realm-gradle-plugin:3.7.2"
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.2.41"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.10"
```
Comment (iamareebjamal) [inline build.gradle]: Irrelevant changes

Comment (NileshPranami) [reply build.gradle]: Yaa this is irrelevant. Need not to be change.

File: app/src/main/java/org/fossasia/susi/ai/skills/SkillsActivity.kt (hunk 1 above)
Comment (NileshPranami) [reply app/src/main/java/org/fossasia/susi/ai/skills/SkillsActivity.kt]: Thanks for your review sir.
I will try to make it better, If you have any suggestions for me then please tell me.

**Comment by NileshPranami** [comment]
Closing this PR because of some build error in codes.

**Pull request closed without merging**